dataline-singer/src/main/resources/json/SingerMessage.json

```diff
@@ -4,13 +4,13 @@
   "title": "SingerMessage",
   "description": "This is a compromise. Java codegen doesn't support union types. So we need to just make a flat json schema that looks like at of the possible permutations of a singer message. See the description for each field to see which message type that field belongs to.",
   "type": "object",
-  "additionalProperties": false,
+  "additionalProperties": true,
   "required": ["type"],
   "properties": {
     "type": {
       "description": "record, schema, and state message: used to determine record type. used in all singer messages.",
       "type": "string",
-      "enum": ["RECORD", "SCHEMA", "STATE"]
+      "enum": ["RECORD", "SCHEMA", "STATE", "ACTIVATE_VERSION"]
```

Comment:
what is that?

Reply:
See comment on the file -- it's an undocumented message type in singer that is widely used by DBs apparently